.gitlab-ci.yml

```diff
@@ -1089,11 +1105,20 @@ build_mender-artifact:
     - cd ${WORKSPACE}
     - tar -xf /tmp/workspace.tar.gz
     - mv /tmp/build_revisions.env .
+
+    # Post job status
+    - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
+
     # cd into component path
     - cd ${WORKSPACE}/go/src/github.com/mendersoftware/mender-artifact
   script:
     - make build-natives-contained
     - cp mender-artifact-* $CI_PROJECT_DIR/
+
+  after_script:
+    # Post job status
```

Comment:
You are missing here defining WORKSPACE.  I think are the calls to `$WORKSPACE/integration/extra/release_tool.py ` from inside the script which are returning "not found".

You should be able to fix by adding:
```
- export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
```

Reply:
While it's probably true this was missing, it doesn't seem to be the problem, because it happens much earlier that `after_script`.

Reply:
You were right here.